### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is "null" for "search" button and "csv file" button for variable records in variable explorer pane

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/RTVS#3341]
**Platform:**
Windows 10 RS1
Visual Studio Enterprise 2017 Preview
Version: 15.0.26340.0

**Repro Steps:**
1. Launch Visual Studio
2. Hit "Alt+R", to open "Rtools" menu
3. Using arrow keys navigate to "Data Science Settings" menu item and hit enter
4. In the "R Interactive" pane, enter the command for ex., "x<-iris" and hit enter to get the variable in the "Variable Explorer"
5. Navigate to the created variable in "Variable Explorer" pane
6. Tab to "Show item details" and "Open in CSV" buttons in the created variable and observe the name property is inspect

**Expected Result:**
Name property should be "Show item details" and "Open in CSV app" for the respective buttons on the created variable

**Note:**
"Show item details" and "Open in CSV app" is defined in help text instead. 

**Actual Result:**
Name property is null for the buttons in the created variable

**Narrator Behavior:**
Narrator announces as "Button button"

**User Impact:**
Screen reader dependent users will not get to know the exact purpose of the buttons if the name property is not defined

![inspect_name property_open in csv button](https://cloud.githubusercontent.com/assets/22956130/24295729/2e9ad2d0-10c2-11e7-8a97-0d30667023b5.jpg)
![inspect_name property_search button](https://cloud.githubusercontent.com/assets/22956130/24295730/2e9b0b7e-10c2-11e7-83f7-35ccfbf9929e.jpg)
![narrator log_name property null for buttons](https://cloud.githubusercontent.com/assets/22956130/24295731/2e9b5aa2-10c2-11e7-8c7b-6ab2330c9e7a.jpg)

**Comment by AlexanderSher** [comment]
PR was submitted to Release/1_0
